aixtend/modules/model.py

```diff
@@ -27,6 +27,11 @@
 import logging
 import traceback
 from requests.adapters import HTTPAdapter, Retry
+from collections import namedtuple
+from typing import List
+
+ModelInfo = namedtuple('ModelInfo', ['name', 'id', 'supplier'])
```

Comment:
Did you consider making all these info objects as classes? What if we have more fields that we need to accommodate in the future? Would a tuple be forwards compatible with more values?

Reply:
yes, this should be modeled as attributes of the class "Model" instead

Reply:
We will add the information to the module as attributes and shift the functional uses(listing etc) inside factories as suggested by @thiago-aixplain